indexing-service/src/main/java/io/druid/indexing/common/config/TaskConfig.java

```diff
@@ -73,9 +74,9 @@ public TaskConfig(
       @JsonProperty("directoryLockTimeout") Period directoryLockTimeout
   )
   {
-    this.baseDir = baseDir == null ? "/tmp" : baseDir;
+    this.baseDir = baseDir == null ? System.getProperty("java.io.tmpdir") : baseDir;
     this.baseTaskDir = new File(defaultDir(baseTaskDir, "persistent/task"));
-    this.hadoopWorkingPath = defaultDir(hadoopWorkingPath, "druid-indexing");
+    this.hadoopWorkingPath = hadoopWorkingPath == null ? "/tmp/druid-indexing" : hadoopWorkingPath;
```

Comment:
should this also default to "java.io.tmpdir"/druid-indexing instead of /tmp/druid-indexing ?